week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
+    
+}
+
+.top{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* border: 2px solid red; */
+    height: 20%;
+    width: 30%;
+    
+    
+}
+
```

p 태그는 자동으로 요소의 위쪽과 아래쪽에 약간의 여백을 추가합니다. 따라서 원활한 CSS 작업을 위해 `margin: 0;`으로 초기화하는 것이 좋습니다. `.score`와 `.text`를 div 태그에서 p 태그로 변경하였으므로, 다음의 코드를 추가합니다.
```suggestion
body .result p {
    margin: 0;
}
```